components/mod-event/EventItem.tsx

```diff
@@ -101,10 +108,12 @@ const Comment = ({
       {/* This is only for legacy actions, new actions won't have these properties for these events */}
       <EventLabels
         header="Added: "
+        // @ts-ignore
```

Comment:
These are only for legacy support for data that is more than a yr old so not too worried about typings here.